bin/conjur-cli/commands/server.rb

```diff
@@ -85,6 +88,16 @@ def load_bootstrap_policy
       ) || exit(($CHILD_STATUS.exitstatus))
     end
 
+    def cleanup_pidfile
+      Process.fork do
+        puts("Cleaning up the pidfile")
+
+        exec("
+          rm -f /opt/conjur-server/tmp/pids/server.pid 
+        ")
```

Comment:
I would suggest first checking if the stale PID file exists, and only print the message and delete it if it does. This will provide more information later if we're reviewing logs for a customer, for example.

We also don't need to exec to delete the file, we should do that just within ruby:

```ruby
pid_file_path = '/opt/conjur-server/tmp/pids/server.pid'
if File.exist?(pid_file_path)
  puts("Removing existing PID file: #{pid_file_path}")
  File.delete(pid_file_path)
end
```